src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+
+  if (days < 3) {
+    return days * cost;
+  } else if (days >= 3 && days < 7) {
+    return days * cost - 20;
+  } else {
+    return days * cost - 50;
+  }
```

Comment:
```suggestion

  return days * cost - 50;
```
Этот `else` не нужен